integrations/webchat/examples/custom-buttons/client/react/public/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Custom buttons - Watson Assistant web chat toolkit</title>
```

Comment:
I don't really understand what you're suggesting. Are you saying that "show me credit cards" is not good enough?

Reply:
The response for custom buttons shows
<img width="367" alt="Screen Shot 2022-08-18 at 9 52 47 AM" src="https://user-images.githubusercontent.com/7141092/185412106-f7a36159-c43f-4495-8bec-d6c91a59f304.png">

I was just suggesting that the line above the buttons "What type of credit card are you interested in?" could have the word "rewards" or "benefits" in it since the buttons listed aren't credit card types but instead kinds of rewards.

Although like I said originally, it's super minor and probably not worth changing.